examples/server/server.cpp

```diff
@@ -217,6 +221,32 @@ struct whisper_print_user_data {
     int progress_prev;
 };
 
+bool convert_to_WAV( std::string& temp_filename, std::string &error_resp) {
+    std::ostringstream cmd_stream;
+    std::string converted_filename_temp = temp_filename + "_temp.wav";
+    cmd_stream << "ffmpeg -i \"" << temp_filename << "\" -ar 16000 -ac 1 -c:a pcm_s16le \"" << converted_filename_temp << "\" 2>&1";
+    std::string cmd = cmd_stream.str();
+
+    int status = std::system(cmd.c_str());
+    if (status != 0) {
+        error_resp = "{\"error\":\"FFmpeg conversion failed. Check if FFmpeg is installed\"}";
```

Comment:
I'm not sure if this is what we want.

We can obtain the exact output error output from system(), but that requires additional code. I can implement that if it makes more sense.

However, it will mostly fail if ffmpeg is not installed.

Reply:
Yes, I think that is enough and will cover most cases.